src/Compilers/CSharp/Test/Symbol/Symbols/DefaultInterfaceImplementationTests.cs

```diff
@@ -14662,23 +14901,40 @@ public virtual int P1
-                // class Test1 : I1
-                Diagnostic(ErrorCode.ERR_ImplicitImplementationOfNonPublicInterfaceMember, "I1").WithArguments("Test1", "I1.P1.get", "Test1.P1.get", "9.0", "10.0").WithLocation(2, 15)
+                expectedIn9: ExpectedDiagnostics(
+                    // (11,9): error CS8704: 'Test1' does not implement interface member 'I1.P1.get'. 'Test1.P1.get' cannot implicitly implement a non-public member in C# 9.0. Please use language version '10.0' or greater.
+                    //         get
+                    Diagnostic(ErrorCode.ERR_ImplicitImplementationOfNonPublicInterfaceMember, "get").WithArguments("Test1", "I1.P1.get", "Test1.P1.get", "9.0", "10.0").WithLocation(11, 9),
+                    // (16,9): error CS8704: 'Test1' does not implement interface member 'I1.P1.set'. 'Test1.P1.set' cannot implicitly implement a non-public member in C# 9.0. Please use language version '10.0' or greater.
+                    //         set
+                    Diagnostic(ErrorCode.ERR_ImplicitImplementationOfNonPublicInterfaceMember, "set").WithArguments("Test1", "I1.P1.set", "Test1.P1.set", "9.0", "10.0").WithLocation(16, 9)
+                    ),
+                expectedAccrossAssemplyBoundaries: ExpectedDiagnostics(
+                    // (11,9): error CS9044: 'Test1' does not implement interface member 'I1.P1.get'. 'Test1.P1.get' cannot implicitly implement an inaccessible member.
+                    //         get
+                    Diagnostic(ErrorCode.ERR_ImplicitImplementationOfInaccessibleInterfaceMember, "get").WithArguments("Test1", "I1.P1.get", "Test1.P1.get").WithLocation(11, 9),
+                    // (16,9): error CS9044: 'Test1' does not implement interface member 'I1.P1.set'. 'Test1.P1.set' cannot implicitly implement an inaccessible member.
+                    //         set
+                    Diagnostic(ErrorCode.ERR_ImplicitImplementationOfInaccessibleInterfaceMember, "set").WithArguments("Test1", "I1.P1.set", "Test1.P1.set").WithLocation(16, 9)
+                    )
                 );
         }
 
         private void ValidatePropertyModifiers_11_02(string source1, string source2,
-                                                  params DiagnosticDescription[] expected)
+                                                     params DiagnosticDescription[] expectedIn9)
+        {
+            ValidatePropertyModifiers_11_02(source1, source2, expectedIn9, expectedAccrossAssemplyBoundaries: Array.Empty<DiagnosticDescription>());
+        }
+
+        private void ValidatePropertyModifiers_11_02(string source1, string source2,
+                                                     DiagnosticDescription[] expectedIn9,
+                                                     params DiagnosticDescription[] expectedAccrossAssemplyBoundaries)
```

Comment:
`expectedAccrossAssemplyBoundaries` -> `expectedAcrossAssemplyBoundaries`

Reply:
Worth doing a find/replace across the entire file, this was copy/pasted a few times.